x-pack/plugins/alerting/server/task_runner/rule_action_helper.ts

```diff
@@ -69,6 +92,10 @@ export const isSummaryActionThrottled = ({
 };
 
 export const generateActionHash = (action?: RuleAction) => {
+  if (action != null && isSystemAction(action)) {
+    return `${action?.actionTypeId || 'no-action-type-id'}:summary`;
```

Comment:
I am not sure if this is the correct way to approach the hash of a system action.